README.md

```diff
@@ -220,6 +222,7 @@ spinnaker:
                   # By default this plugin adds a set of sane default tags to help with observability best practices, you can disable those here
                   # Optional, Default: false
                   defaultTagsDisabled: false
+                  excludedMetrics: ["Metric1","Metric2"] #configure Metrics to exclude
```

Comment:
NOTE:  You're using denyNameStartsWith - it'd be good to call this out as a "excludedMetricsPrefix" OR perhaps change the metric filter to deny on that id ONLY.

Reply:
Changed the name :)